bindings/wasm/src/tangle/client.rs

```diff
@@ -158,6 +158,20 @@ impl WasmClient {
     Ok(promise)
   }
 
+  /// Checks if a message is confirmed by a milestone.
+  #[wasm_bindgen(js_name = isMessageIncluded)]
+  pub fn is_message_included(&self, message_id: &str) -> Result<PromiseBoolean> {
```

Comment:
Prefer camelCase parameter names in Wasm (I know this is not consistent everywhere yet, but it should be).

```suggestion
  #[wasm_bindgen(js_name = isMessageIncluded)]
  #[allow(non_snake_case)]
  pub fn is_message_included(&self, messageId: &str) -> Result<PromiseBoolean> {
```